src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -238,6 +253,14 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
     navigation.navigate("DailyStatisticsScreen");
   };
 
+  const getBarcodeType = (): any[] => {
+    if (selectedIdType.scannerType === "QR")
+      return [BarCodeScanner.Constants.BarCodeType.qr];
+    else if (selectedIdType.scannerType === "CODE_39")
+      return [BarCodeScanner.Constants.BarCodeType.code39];
+    else return [null];
```

Comment:
This removes the notion that there is a default of `code39`, which was true before. Might be better to keep the original logic as a fail-safe!

Let me know what you think!

```suggestion
    if (selectedIdType.scannerType === "QR") {
      return [BarCodeScanner.Constants.BarCodeType.qr];
    } else if (selectedIdType.scannerType === "CODE_39") {
      return [BarCodeScanner.Constants.BarCodeType.code39];
    } else {
      return features?.id.scannerType === "QR"
              ? [BarCodeScanner.Constants.BarCodeType.qr]
              : [BarCodeScanner.Constants.BarCodeType.code39];
    }
```

Reply:
good idea!